src/common/transformations/src/transformations/common_optimizations/mark_precision_sensitive_subgraphs.cpp

```diff
@@ -20,6 +20,7 @@ using namespace std;
 bool ov::pass::MarkPrecisionSensitiveSubgraphs::run_on_model(const std::shared_ptr<ov::Model>& f) {
     deque<Node*> nodes;
     unordered_set<Node*> visited, precision_sensitive_visited;
+    bool rewritten = true;
```

Comment:
I think need use false value. If you use true, rewritten always will be  have true value.

Reply:
This value is never changed inside the function.

Reply:
jk, lets delete this useless variable

Reply:
yes, I have removed this variable as useless and return old state as before changes